[2024-06-09 23:39:28,257][backoff][INFO] - Backing off chat_completion(...) for 0.6s (aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://api.anthropic.com/v1/messages'))
[2024-06-09 23:39:29,132][backoff][INFO] - Backing off chat_completion(...) for 0.0s (aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://api.anthropic.com/v1/messages'))
[2024-06-09 23:39:29,491][backoff][INFO] - Backing off chat_completion(...) for 3.2s (aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://api.anthropic.com/v1/messages'))
[2024-06-09 23:39:32,997][backoff][INFO] - Backing off chat_completion(...) for 0.7s (aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://api.anthropic.com/v1/messages'))
[2024-06-09 23:39:33,930][backoff][INFO] - Backing off chat_completion(...) for 12.2s (aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://api.anthropic.com/v1/messages'))
[2024-06-09 23:39:46,370][backoff][INFO] - Backing off chat_completion(...) for 25.3s (aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://api.anthropic.com/v1/messages'))
[2024-06-09 23:40:12,007][backoff][INFO] - Backing off chat_completion(...) for 37.8s (aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://api.anthropic.com/v1/messages'))
[2024-06-09 23:40:50,334][backoff][ERROR] - Giving up chat_completion(...) after 8 tries (aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://api.anthropic.com/v1/messages'))
